DrrImageComputation/Logic/vtkSlicerDrrImageComputationLogic.h

```diff
@@ -198,6 +199,8 @@ class VTK_SLICER_DRRIMAGECOMPUTATION_MODULE_LOGIC_EXPORT vtkSlicerDrrImageComput
   vtkSlicerCLIModuleLogic* PlastimatchDRRComputationLogic;
   /// Beams logic instance
   vtkSlicerBeamsModuleLogic* BeamsLogic;
+  /// Rooms eye view logic instance
+  vtkSlicerRoomsEyeViewModuleLogic* REVLogic;
```

Comment:
I think these can be accessed dynamically in every function using the new feature in the slicer application logic class.